### PR TITLE
fix #41087 timesofindia.indiatimes.com

### LINKED ISSUE #41087: timesofindia.indiatimes.com
### Issue URL (Ads)
[https://timesofindia.indiatimes.com/india/mild-earthquake-tremors-felt-in-delhi-ncr/articleshow/71276697.cms](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Ftimesofindia.indiatimes.com%2Findia%2Fmild-earthquake-tremors-felt-in-delhi-ncr%2Farticleshow%2F71276697.cms)
### Comment
Username: @sangalaviral
> 
> --
> The information below is automatically generated. Do not delete it.
> 
> Referrer:
> https://news.google.com/
> 
> Custom userscripts:
> https://userscripts.adtidy.org/release/popup-blocker/2.5/popupblocker.user.js
> https://userscripts.adtidy.org/release/assistant/4.3/assistant.user.js
> https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
Platform: | Win
AdGuard version: | 7.2.2936.0
Browser: | Chrome
AdGuard driver: | WFP
Filters: | AdGuard Annoyances,<br>AdGuard Base,<br>AdGuard Tracking Protection,<br>AdGuard Social Media

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/41087

### CODE REVIEW

File: EnglishFilter/sections/foreign.txt [hunk 1]
```diff
@@ -2457,6 +2457,8 @@ bb.is$$div[class="banner"]
 ratopati.com##div[style*="max-height:150px;overflow:hidden;"]
 ! https://github.com/AdguardTeam/AdguardFilters/issues/34513
 jagran.com##.article-ctn
+! https://github.com/AdguardTeam/AdguardFilters/issues/41087
+timesofindia.indiatimes.com##div[data-primead="articleshow665X100"]
```
Comment (BlazDT) [inline EnglishFilter/sections/foreign.txt]: Too specific, please check if ##div[data-primead] breaks something.

Comment (zloyden) [reply EnglishFilter/sections/foreign.txt]: > please check if ##div[data-primead] breaks something.

Looks fine on my end. Also added rule to hide third-party news.

**Comment by AdamWr** [comment]
I see more ads/ad leftovers, so please maybe add also:
```
timesofindia.indiatimes.com##.toi-amazon
timesofindia.indiatimes.com##.asprime1
timesofindia.indiatimes.com###reverseads
timesofindia.indiatimes.com##.mid_ads
timesofindia.indiatimes.com##.fromaroundwebrhs
timesofindia.indiatimes.com#?#.as_article > div.article_content > div[class*=" "]:has(> div > div > a[onclick])
timesofindia.indiatimes.com#?#.sidebar > div[class*=" "]:has(> div > div > a[onclick])
timesofindia.indiatimes.com#?#.main-content > div.article_content~div[class*=" "]:has(> div > div > a[onclick])
```
I think that we can remove the `[id]` part from this rule:
`timesofindia.indiatimes.com#?#.main-content > div.article_content~div[class*=" "][id]:has(> div > div > a[onclick])`

<details><summary>Screenshot</summary>

image

image

image

image

</details>

**Comment by zloyden** [comment]
@AdamWr these ads still not reproduced on my end. If that rules works, I can add them.

**Comment by AdamWr** [comment]
Please also remove this rule:
https://github.com/AdguardTeam/AdguardFilters/blob/09d302e4180cc352c212fe9d8ed31cd86fe83875/EnglishFilter/sections/css_extended.txt#L241
This is the same rule, but without `[id]` part - https://github.com/AdguardTeam/AdguardFilters/commit/09d302e4180cc352c212fe9d8ed31cd86fe83875#diff-ca3b6170ccb0415a0a5b76e4bf9e1acaR609

**Comment by Alex-302** [comment]
also social shares 
image

**Review by AdamWr — APPROVED** [review]

**Pull request merged**